src/AutoRest.CSharp/Mgmt/Output/MgmtLongRunningOperation.cs

```diff
@@ -19,25 +18,15 @@ namespace AutoRest.CSharp.Mgmt.Output
     /// </summary>
     internal class MgmtLongRunningOperation : LongRunningOperation
     {
```

Comment:
Why is it that there are changes to management plane as part of this PR?

Reply:
SubClients have broken the general naming pattern for clients. Previously, all clients had independent names. Now, LLC client may have different name depending on either it is a top level client or subclient.